app/main.development.js

```diff
@@ -873,6 +873,19 @@ app.on("ready", async () => {
     height: 1000,
     icon: __dirname + "/dcrdex.png"
   });
+
+  mainWindow.webContents.session.setDevicePermissionHandler((details) => {
+    // Allow Ledger devices which share a unique vendor ID.
+    if (
+      details.deviceType === "usb" &&
+      details.origin === "http://localhost:3000" &&
+      details.device &&
+      details.device.vendorId === 0x2c97
+    ) {
+      return true;
+    }
+    return false;
+  });
```

Comment:
This seems all to be that is needed for the dev environment and linux. Unsure yet if this is sufficient.

Reply:
After building, this is not enough. This does not work when packaged.